.maintain/README_zh_CN.md

```diff
@@ -30,193 +30,31 @@ cargo test
-### 使用 srtool
-
-
-关于 srtool 的使用方式, 这里进行简单的说明
-
-> 更详细的使用说明可以参考 [chevdor/srtool/README.adoc](https://gitlab.com/chevdor/srtool), 因为 Docker hub 中没有对应的镜像, 因此也可以去看 [chevdor/srtool/README.adoc](https://gitlab.com/chevdor/srtool/-/blob/master/README.adoc)
-
+2. 修改 [release-template.md](./release-template.md)
+   Github release page 页面所显示的内容是根据此模板而生成的. 如果需要请编辑此模板.
 
-首先, srtool 支持的命令包括
+## subwasm
 
-- `help`: 查看帮助
-- `version`: srtool 版本
-- `info`: 在运行构建之前查看一些系统信息
-- `build`: 进行 runtime 编译
-
-
-由于 srtool 发布了 docker 镜像, 可以利用 docker 的运作模式配置 bash 等相关特性来构建一个可执行的脚本文件
-
-例如官方所发布的一个简单的别名脚本
-
-```bash
-export RUSTC_VERSION=nightly-2021-03-15;
-export PACKAGE=kusama-runtime;
-alias srtool='docker run --rm -it -e PACKAGE=$PACKAGE -v $PWD:/build -v $TMPDIR/cargo:/cargo-home chevdor/srtool:$RUSTC_VERSION'
-```
-
-这里的含义是, 将目前所在路径挂载到 docker 容器中, 并使用指定的 rust 版本来对当前 runtime 进行编译. 其中 `PACKAGE` 就是需要编译的 runtime 名称
-
-
-编译使用
-
-```bash
-srtool build
-```
-
-进行编译后的输出后内容类似于下方
-
-
-```text
-🧰 Substrate Runtime Toolbox - srtool v0.9.6 🧰
-          - by Chevdor -
-🏗  Building polkadot-runtime as release using rustc 1.49.0-nightly (fd542592f 2020-10-26)
-⏳ That can take a little while, be patient... subsequent builds will be faster.
-Since you have to wait a little, you may want to learn more about Substrate runtimes:
-https://substrate.dev/docs/en/#architecture
-
-    Finished release [optimized] target(s) in 37.43s
-
-real  0m37.931s
-user  0m1.560s
-sys 0m3.220s
-✨ Your Substrate WASM Runtime is ready! ✨
-Summary:
-Used rustc nightly-2021-03-15 (4560ea788 2019-11-04)
-Wasm     : ./[some path]/polkadot_runtime.compact.wasm
-Content  : 0x0061736d0100000001a4022b60037f7f...3435663020323031392d31322d303429
-Size     : 1.1M
-Proposal : 0x5931690e71e9d3d9f04a43d8c15e45e0968e563858dd87ad6485b2368a286a8f
-SHA256   : 0xd5930520676994fc55a29c547f0159ea860cb46edd710a5be35e62565af1ad8b
-```
-
-同时支持以 json 格式输出
-
-```bash
-srtool build --json
-```
-
-
-```text
-{
-  "gen": "srtool",
-  "rustc": "rustc 1.41.0-nightly (ae1b871cc 2019-12-06)",
-  "wasm": "./target/srtool/release/wbuild/kusama-runtime/kusama_runtime.compact.wasm",
-  "size": "1205052",
-  "pkg": "kusama-runtime",
-  "prop": "0x5931690e71e9d3d9f04a43d8c15e45e0968e563858dd87ad6485b2368a286a8f",
-  "sha256": "0xd93126c814f8366b651e425e34390212a98f8e77a8b73f9e1d2b07fc229a25f1",
-  "tmsp": "2020-01-14T10:15:28Z"
-}
-```
-
-这里需要注意的是 `Proposal` 字段, 这个值应该要和 apps 中发布时的值相同
+Darwinia 直接采用 subwasm 来进行 runtime wasm proposal hash 计算, 采用与 srtool 相同的环境, 在计算完成后, 会输出变量到 release page 模板中, 视需要输出到页面, 同时也会将计算出的 json 存储为文件上传到 release page.
 
 ### 验证
 
-如果想要自行验证 wasm hash, 可以通过 srtool 来进行 darwinia/crab runtime.
-
-> 如上方阐述, 存在 [paritytech/srtool](https://hub.docker.com/r/paritytech/srtool) 以及 [chevdor/srtool](https://gitlab.com/chevdor/srtool) 两个库, Darwinia 使用的是 paritytech/srtool, 如果要自行验证也应该使用 paritytech/srtool, 保证自行编译使用的版本和 Darwinia 所使用的相同即可.
-
-为了能够正确编译, 你需要安装以下软件
+如果想要自行验证 wasm hash, 可以通过 polkadot 钱包在上传 wasm 后会得到一个 proposal hash, 然后与 CI 发布的进行对比.
```

Comment:
Better correcting all the capitalization in this doc, such as "Polkadot", "GitHub", etc.